src/resolvers/http.ts

```diff
@@ -0,0 +1,16 @@
+import { Resolver } from '@stoplight/json-ref-resolver';
+import unfetch from 'unfetch';
+
+export const httpReader = {
+  async read(ref: any) {
+    return unfetch(String(ref)).then(r => r.text);
```

Comment:
code-style: you could use `await` here.

Reply:
Yeah or we can just remove the async from line 5 :).

Reply:
Yeah, you are also missing a function call 
```suggestion
    return unfetch(String(ref)).then(r => r.text());
```
Will fix it.